### ISSUE TITLE
support for Visual Studio 2022 

### ISSUE THREAD

**Issue by Gandalf-Le-Dev** [Hiberbee/code-highlight-themes#94]
Can you make an official release of the theme on Visual Studio 2022 with the VSCode theme converter https://github.com/microsoft/theme-converter-for-vs for example (not sure if the result will be accurate)

Thanks for this awesome theme I love it.

**Comment by vladyslavvolkov** [comment]
Yep, will figure out how good that is. Frankly, not a every-day VS user, so will take week or sort of to come back
